routers/routes/routes.go

```diff
@@ -513,8 +513,9 @@ func RegisterRoutes(m *macaron.Macaron) {
 		})
 	}, ignSignIn)
 
-	m.Group("", func() {
-		m.Post("/attachments", repo.UploadAttachment)
+	m.Group("/attachments", func() {
+		m.Post("", repo.UploadAttachment)
+		m.Post("/delete", repo.DeleteAttachment)
```

Comment:
Just asking on a wider specs, why we don't use DELETE http command? The POST /delete path seems to be what is commonly used across gitea path but it seems we could use the more REST like pattern. (m.Delete("/:uuid", repo.DeleteAttachment))

@blueworrybear the code is fine, I just jump on the occasion on wider design pattern.

Reply:
`DELETE` is used only api not in UI routes

Reply:
@sapk 
I have the same question when I started updating the code. In fact, I tried to used `DELETE` request in the very begin.

However, after I start editing `index.js`, I found that `$.ajax` cannot send data with `DELETE`.
After studying, it seems like `DELETE` request may not has `body` ([Methods/DELETE](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/DELETE)). Besides, there is not `$.delete` shortcut in **jquery**. Therefore I refer to the coding for deleting release attachments which uses `$.post`.

In summary, I guess the reason not to use `DELETE` in UI route would be:

- No **jquery** shortcut
- Cannot send data with request body